types/object-map/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for object-map 1.0
+// Project: https://github.com/xixixao/object-map
+// Definitions by: Wolfgang Faust <https://github.com/wolfgang42>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function objectMap<TInput, TOutput>(
+	target: {[k: string]: TInput},
+	callback: (currentValue: TInput, key: string, object: {[k: string]: TInput}) => TOutput,
+	thisArg?: any
```

Comment:
If you really want to support a `thisArg` you might want an overload using `callback: (this: TThis, ...) => TOutput`.

Reply:
Cool, I didn't realize that was possible, I didn't see anything about declaring `this` when I looked in the TS documentation. I'll push a new version in a few minutes. Thanks!